kore/app/exec/Main.hs

```diff
@@ -558,20 +562,14 @@ main = do
 
 mainWithOptions :: KoreExecOptions -> IO ()
 mainWithOptions execOptions = do
-    let KoreExecOptions
-            { koreLogOptions
-            , koreSolverOptions
-            , bugReportOption
-            }
-          = execOptions
+    let KoreExecOptions { koreSolverOptions, bugReportOption, outputFileName } = execOptions
     ensureSmtPreludeExists koreSolverOptions
     exitCode <-
         withBugReport Main.exeName bugReportOption $ \tmpDir -> do
             writeOptionsAndKoreFiles tmpDir execOptions
-            go <* warnIfLowProductivity
-                & handle handleWithConfiguration
-                & handle handleSomeException
-                & runKoreLog tmpDir koreLogOptions
+            e <- code tmpDir
+            for_ outputFileName $ flip copyFile (tmpDir </> "result.kore")
```

Comment:
This will only capture the output if it is being written to a file already. It will not capture the standard output. We should write the output to a temporary file _first_, and copy the file or print it to standard output afterward.